### ISSUE TITLE
Add link to toggle fieldtypes

### ISSUE THREAD

**Issue by aerni** [statamic/ideas#835]
What's the suggested way of providing a link to the `inline_label` of toggle fieldtypes for forms? I often use this field for things like terms and conditions that need an inline link. I always have to work around it. This just doesn't feel right:

```yaml
inline_label: 'I accept the <a href="https://privacy.com" class="underline hover:opacity-60">terms and conditions</a>'
```

**Comment by jesseleite** [comment]
It's the only option at the moment. Open to ideas!

**Comment by aerni** [comment]
What about making that inline label a Bard field with basic text styling options? 

Or remove the inline label from the form submission view to prevent this ugliness:

<img width="987" alt="Bildschirmfoto 2022-07-29 um 12 20 42" src="https://user-images.githubusercontent.com/23167701/181802215-03ec31ca-66e5-441b-874e-67b1f6fd6bcc.png">

**Comment by jesseleite** [comment]
Not sure how I feel about a full bard for inline/single-line text, but removing the inline label from the submission view makes sense; It's just extra context for the user on the front end.

**Comment by aerni** [comment]
I agree, I don't know how I feel about it either 😅 I believe removing is the better approach for now.

**Comment by jasonvarga** [comment]
It should accept markdown.

**Comment by aerni** [comment]
The master has spoken.
